sycl/doc/syclcompat/README.md

```diff
@@ -971,6 +1060,87 @@ T atomic_compare_exchange_strong(
     T *addr, T expected, T desired,
     sycl::memory_order success = sycl::memory_order::relaxed,
     sycl::memory_order fail = sycl::memory_order::relaxed);
+template <typename T, typename T2, typename ValueV,
+          sycl::access::address_space addressSpace =
+              sycl::access::address_space::global_space,
+          sycl::memory_order memoryOrder = sycl::memory_order::relaxed,
+          sycl::memory_scope memoryScope = sycl::memory_scope::device>
+T atomic_compare_exchange_strong(
+    T *addr, T2 expected, ValueV desired,
+    sycl::memory_order success = sycl::memory_order::relaxed,
+    sycl::memory_order fail = sycl::memory_order::relaxed);
+
+} // namespace syclcompat
+```
+
+The `atomic` class offers another interface to atomic operations, providing the
+`store`, `load`, `exchange`, `compare_and_exchange`, `fetch_add`, and
+`fetch_sub` operations.
+
```

Comment:
I think the content from here onwards is already merged in another PR, so might need to rebase.